### PR TITLE
fix cve normalizer

### PR DESCRIPTION
**Pull request by noamblitz**
### Changes
More than half of CVEs didn't get their data hydrated because of several bugs in the normalizer. This pr fixes all (hopefully).

### Demo
image

---

### Code Checklist
- [ ] All the commits in this PR are properly PGP-signed and verified.
- [ ] This PR only contains functionality relevant to the issue; tickets have been created for newly discovered issues.
- [ ] I have written unit tests for the changes or fixes I made.
- [ ] For any non-trivial functionality, I have added integration and/or end-to-end tests.
- [ ] I have performed a self-review of my code and refactored it to the best of my abilities.

### Communication
- [ ] I have informed others of any required `.env` changes files if required and changed the `.env-dist` accordingly.
- [ ] I have made corresponding changes to the documentation, if necessary.
- [ ] I have included comments in the code to elaborate on what is not self-evident from the code itself, including references to issues and discussions online, or implicit behavior of an interface.

---
## Checklist for code reviewers:
Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---
## Checklist for QA:
Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.

### CODE REVIEW

File: boefjes/boefjes/plugins/kat_cve_finding_types/normalize.py [hunk 1]
```diff
@@ -54,7 +55,9 @@ def run(normalizer_meta: NormalizerMeta, raw: Union[bytes, str]) -> Iterable[OOI
 
     yield CVEFindingType(
         id=cve_finding_type_id,
-        description=english_description["value"],
+        description=unicodedata.normalize("NFKC", english_description["value"]).translate(
+            str.maketrans("‘’“”", "''\"\"")
```
Comment (dekkers) [inline boefjes/boefjes/plugins/kat_cve_finding_types/normalize.py:59]: Why is this necessary? We should be able to save the description if it has unicode characters in it?

Comment (noamblitz) [reply boefjes/boefjes/plugins/kat_cve_finding_types/normalize.py:59]: Dont exactly know why that doesnt work, but this seems like a harmless fix. Then I can move on with more urgent things.

Comment (dekkers) [reply boefjes/boefjes/plugins/kat_cve_finding_types/normalize.py:59]: As discussed today, I'm going to look into the root cause of this problem. I've assigned the PR to me.

**Comment by originalsouth** [comment]
Initial bug of allowing OOIs with Unicode seems to be in [`model_dump_json()`](https://docs.pydantic.dev/latest/concepts/serialization/) not escaping the Unicode (like a custom `json.dump()` would) this results in the connector, panicking causing a 422

**Comment by originalsouth** [comment]
> Initial bug of allowing OOIs with Unicode seems to be in [`model_dump_json()`](https://docs.pydantic.dev/latest/concepts/serialization/) not escaping the Unicode (like a custom `json.dump()` would) this results in the connector, panicking causing a 422

Case study:
```
#!/usr/bin/env python
import json
import pydantic

class Cat(pydantic.BaseModel):
    name: str

cat = Cat(name = "kittie")
print(cat.model_dump_json())
print(json.dumps({"name": "kittie"}))

cat = Cat(name = "🐱")
print(cat.model_dump_json())
print(json.dumps({"name": "🐱"}))
```

Results in:
```
% python test.py
{"name":"kittie"}
{"name": "kittie"}
{"name":"🐱"}
{"name": "\ud83d\udc31"}
```

**Comment by underdarknl** [comment]
We are going to fix the connector / otopoes, so this pr is to be closed.
